Assets/MainWindow.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MainWindow : MonoBehaviour
```

Comment:
Maybe we don't need this file until we create a story for this

Reply:
oops, got too involved in the tutorial. :)